### ISSUE TITLE
About the training data

### ISSUE THREAD

**Issue by sunshantong** [DC1991/G2L_Net#8]
Hi, thank you for your great work.  Will you provide the completed training dataset?  Or how do I generate it myself? 
Thank you very much.

**Comment by DC1991** [comment]
@sunshantong Hi, thanks for your interest in our paper and code. I have described how to generate point-wise label with the object 3D model in the paper (in the experiment section). I borrow the render code from this git (https://github.com/thodan/bop_toolkit): render the 3D object model to the scene with GT pose, and label the point (transformed from depth with known camera parameters) less than distance threshold to the rendered 3D model as positive point.

**Comment by sunshantong** [comment]
Thank you for your quick reply.  Sorry for my negligence.  I'll render it in this way.  Thank you very much.
